Extension/package.json

```diff
@@ -4303,7 +4303,7 @@
                       },
                       {
                         "type": "boolean",
-                        "description": "%c_cpp.debuggers.logging.engineLogging.description%",
+                        "description": "%c_cpp.debuggers.logging.natvisDiagnostics.description%",
```

Comment:
Did you run `yarn generateOptionsSchema`?  I believe there are two entries in package.json that should be updated.

Reply:
I only saw one from the issue

Reply:
![image](https://user-images.githubusercontent.com/111317156/217345353-c2cf9638-491b-4639-9d74-f0a2810611f6.png)

Reply:
Is this resolved?

Reply:
We only have this supported for MIEngine `cppdbg`. Not `cppvsdbg`.

Reply:
This can be resolved. The Options schema fixed the boolean description and there are two instances of the boolean being updated. 

Line 3518 and this.